pkg/cmd/cli/cmd/edit.go

```diff
@@ -195,6 +195,13 @@ func RunEdit(fullName string, f *clientcmd.Factory, out io.Writer, cmd *cobra.Co
 			continue
 		}
 
+		visitor := resource.NewFlattenListVisitor(updates, rmap)
+
+		// need to make sure the original namespace wasn't changed while editing
+		if err = visitor.Visit(resource.RequireNamespace(cmdNamespace)); err != nil {
```

Comment:
If a user has made edits, they'll be lost here.  Need to do the same thing to preserve the user's edits.